lib/conjur/policy/yaml/handler.rb

```diff
@@ -202,7 +202,7 @@ class Mapping < Base
           
           def initialize parent, anchor, type
             super parent, anchor
-            
+            @member_count = Set.new()
```

Comment:
Can we name this something like `existing_members` or something similar, rather than `count`?

Reply:
will do

Reply:
+1 for tests :)

Reply:
@oburstein-hub you said to me in person that this will fix more than the `members` issue. Can you provide an example? And if so, maybe the name `existing_members` is too specific. Knowing the general case will help to choose the best name.

Reply:
I think existing_members is still a good name because members not necessarily mean string "members", but it can be members of group - and this is the meaning in this case.

The example is  of what I said to you in person is as follows:
Loading policy.yaml with the below data also will fail because of duplicate role attribute
```
- !host host1
- !host host2

- !variable secret
- !group secret-consumers

- !permit
  role: !group secret-consumers
  role: !variable secret
  ```

Reply:
In this case i think that `existing_attributes` better tells the story.